middleware/index.ts

```diff
@@ -305,25 +294,21 @@ function createLogger(requestID?: string): Logger {
 
 const userFn = getUserFn();
 
-export default async function systemFn(message: object): Promise<object> {
+export default async function systemFn(message: any): Promise<any> {
     const payload = message['payload'];
 
-    // Remap headers to a standard JS object
+    // Remap riff headers to a standard JS object
     const headers = message['headers'].toRiffHeaders();
     Object.keys(headers).map((key: string) => {headers[key] = message['headers'].getValue(key)});
 
-    const requestId = headers['ce-id'] || headers['x-request-id'];
+    const requestId = headers['Ce-Id'] || headers['X-Request-Id'];
```

Comment:
If these are coming directly from an http call, we'd need to do some case-insensitive searching for these two headers since http headers are supposed to base case-insensitive (https://tools.ietf.org/html/rfc7230#section-3.2) or does the RIFF invoker camel-case HTTP headers like this?

Reply:
@dhagberg-sf the riff `http-streaming` adapter converts headers into this camel-case style.

Reply:
If we go down that path, we'd still have to do the lower-casing of header keys at the time `header` is constructed/populated.  This seems ok for the current PR but something we could look at in a follow-in PR.  OK to leave this way for now.